### DISCUSSION TITLE
Database DDL

### DISCUSSION THREAD

**Discussion by davidsoles** [apache/netbeans#3586] [Q&A]
Hello,
I'm trying to generate the DDL for a stored procedure object in my database but I can't find the option. Can someone tell me if this is possible from NetBeans 12.6? Thanks.

**Comment by Chris2011** [comment]
Afaik this is not possible. I wanted to implement the same feature as for JetBrains DataGrip and it is still in my code somwhere but needs to be nailed down. But maybe someone else can help here.
